### DISCUSSION TITLE
Discussions / code snippets from slack

### DISCUSSION THREAD

**Discussion by dargs** [museumsvictoria/nodel#295] [General]
- extending console from 200 lines - for protocol debugging. 
This was talked about 6-12months ago in slack and is now gone. Justin had a hack to allow 200+ lines of console history.

**Comment by paxofsnax** [comment]
is it /ipadress:port/REST/NodeFolder/nodename/logs?max=200

**Reply by dargs** [reply]
Hey Richard.. mmm. nah don't think so, although that REST call could come in handy !

For memory @justparking had a way of making the actual Console screen show more than 200 lines.
